parquet-benchmarks/src/main/java/org/apache/parquet/benchmarks/BenchmarkConstants.java

```diff
@@ -39,4 +39,6 @@
   public static final int PAGE_SIZE_8M = 8 * 1024 * 1024;
 
   public static final int DICT_PAGE_SIZE = 512;
+
+  public static final int SIXTEEN_THREADS = 16;
```

Comment:
If we were to change this value, we would have to rename the constant as well. I would suggest using a more generic name instead, like THREAD_COUNT.